player/history_heuristic/history_heuristic_player.py

```diff
@@ -0,0 +1,24 @@
+from game_logic.igame_state import IGameState
+from player.history_heuristic.history_heuristic_configuration import HistoryHeuristicConfiguration
+from player.history_heuristic.history_heuristic_node import HistoryHeuristicNode
+from player.mcts.mcts_player import MctsPlayer
+
+
+class HistoryHeuristicPlayer(MctsPlayer):
+    def __init__(self, history_heuristic_configuration: HistoryHeuristicConfiguration, player_name="HistoryHeuristic"):
+        super().__init__(history_heuristic_configuration)
```

Comment:
is it working? I guess MctsPlayer won't accept HistoryHeuristicConfiguration

Reply:
Yep, its working. `HistoryHeuristicConfiguration` extends `MctsConfiguration`, so it can also be treated as regular `MctsConfiguration`.